1.

ages = 6, 8, 8, 10, 11, 12, 14

mean = (6+8+8+10+11+12+14)/7 = 9.86
median = 10
mode = 8
variance = sum((x - mean) ** 2) / (n - 1)
         = sum((x - 9.86) ** 2) / (6)
standard deviation = v ** 0.5
standard error = s / (n ** 0.5)
               = s / (7 ** 0.5)

In [1]:
import pandas as pd
import numpy as np
import statistics

In [2]:
df_brady = pd.DataFrame()
df_brady['age'] = [6, 8, 8, 10, 11, 12, 14]

In [6]:
print("mean: ",np.mean(df_brady['age']))
print("median",np.median(df_brady['age']))
print("mode: ",statistics.mode(df_brady['age']))
print("variance: ", df_brady['age'].var())
print("standard deviation: ",np.std(df_brady['age'], ddof=1))
print("standard error: ",np.std(df_brady['age'] ,ddof=1) / np.sqrt(len(df_brady['age'])))

mean:  9.857142857142858
median 10.0
mode:  8
variance:  7.476190476190475
standard deviation:  2.734262327610589
standard error:  1.0334540197243192


2.

estimate of central tendency: 9.857142857142858
estimate of variance: 2.734262327610589

3.

What changes if Cindy goes from 6 to 7?

In [7]:
df_brady['age'] = [7, 8, 8, 10, 11, 12, 14]
print("mean: ",np.mean(df_brady['age']))
print("median",np.median(df_brady['age']))
print("mode: ",statistics.mode(df_brady['age']))
print("variance: ", df_brady['age'].var())
print("standard deviation: ",np.std(df_brady['age'], ddof=1))
print("standard error: ",np.std(df_brady['age'] ,ddof=1) / np.sqrt(len(df_brady['age'])))

mean:  10.0
median 10.0
mode:  8
variance:  6.333333333333333
standard deviation:  2.516611478423583
standard error:  0.9511897312113418


If Cindy changes from 6 to 7, then all the numbers get closer together. 
The mean goes up and variance, std, and std error get lower.

4.

What changes if one of the 8's is replaced with a 1?
Does this change the central tendency or variance estimation methods?

In [10]:
df_brady['age'] = [7, 1, 8, 10, 11, 12, 14]
print("mean: ",np.mean(df_brady['age']))
print("median",np.median(df_brady['age']))
# print("mode: ",statistics.mode(df_brady['age']))
print("variance: ", df_brady['age'].var())
print("standard deviation: ",np.std(df_brady['age'], ddof=1))
print("standard error: ",np.std(df_brady['age'] ,ddof=1) / np.sqrt(len(df_brady['age'])))

mean:  9.0
median 10.0
variance:  18.0
standard deviation:  4.242640687119285
standard error:  1.6035674514745462


Yes, the central tendency and variance estimation will be very different because 8 was close to the mean.  But changing it to a 1 will throw everything way off. 

5.

20%, 23%, 17%, 5%, were fans
What total percentage were fans?

In [14]:
df_rating = pd.DataFrame()
df_rating['approval'] = [20, 23, 17, 5]
print("The overal average percentage of fans was",np.median(df_rating['approval']))



The overal average percentage of fans was 18.5


In [22]:
df_adj_rating = pd.DataFrame()
df_adj_rating['approval'] = [20, 23, 17]
print("The overal average percentage of fans from non-special intrest sources was",np.median(df_adj_rating['approval']))


The overal average percentage of fans from non-special intrest sources was 20.0
